changelog/unreleased/features/6833.md

```diff
@@ -0,0 +1,14 @@
+- Added `RoadComponent.language` value.
+- :warning: Changed `EHorizonEdgeMetadata.names` class from `RoadName` to `RoadComponent`.
```

Comment:
```suggestion
- :warning: Changed `EHorizonEdgeMetadata.names` class from `RoadName` to `RoadComponent`.
- Added `RoadComponent.language` value.
```
nit, this way it will be easier to read.

Reply:
I see we're done with CI, in this case we can skip this. We can always address when composing final changelog.